apps/documentation/src/pages/ressurser/aktorkart.mdx

```diff
@@ -1,18 +1,136 @@
 ---
 title: Aktørkart
-description: aktørkart
+description: En metode for å kartlegge aktører som blir berørt av et prosjekt
 route: /ressurser/aktorkart
 parent: ressurser
 menu: Workshopmaler
-order: 2
 ---
 
+import { graphql } from 'gatsby';
 import { SEO } from '@components/seo/SEO';
+import figma from './figma.svg'
+import mural from './mural.svg'
+import { DetailsList }  from '@components/Pages/DetailsList';
+import { GuidelinesBlock }  from '@components/Pages/GuidelinesBlock';
+import { TimeglassIcon, UsersIcon, ClockIcon } from '@entur/icons';
+import { NavigationCard } from '@entur/layout';
 
 export const Head = ( props ) => {
   return <SEO title={props.pageContext.frontmatter.title} description={props.pageContext.frontmatter.description} pathname={props.pageContext.frontmatter.route} />
 }
 
+export const query = graphql`
+query wsQuery {
+  files: allFile(filter: {sourceInstanceName: {eq: "media"}, relativeDirectory:  {glob: "images/ressurser/workshopmaler/**"}, extension: {eq: "png"}}) {
```

Comment:
I disse spørringene tror jeg vi kunne hentet ut filen direkte gjennom å filtrere på navnet den
```
query wsQuery {
    file(filter: *en filtermetode på navn eller path inneholder* …
```
i stedet for å hente ut alle filer og filtrere, men det ser ut til at det ikke er store performances issues og vi skal uansett bytte dette til Sanity-basert snart, så du får bestemme selv om du vil gå over optimaliseringen eller ikke.